flutter_local_notifications/windows/registration.cpp

```diff
@@ -0,0 +1,268 @@
+// Huge credit to these StackOverflow answers:
+// https://stackoverflow.com/questions/51947833/activation-from-c-winrt-dll
+// https://stackoverflow.com/questions/67005337/how-works-notifications-on-windows-registry-no-shortlink
+
+#include "registration.h"
+#include "include/flutter_local_notifications/flutter_local_notifications_plugin.h"
+#include "include/flutter_local_notifications/methods.h"
+
+#include <atlbase.h>
+#include <atlconv.h>
+#include <shlobj.h>
+#include <propvarutil.h>
+#include <propkey.h>
+#include <NotificationActivationCallback.h>
+#include <winrt/Windows.UI.Notifications.h>
+#include <winrt/Windows.Data.Xml.Dom.h>
+
+#include <iostream>
+#include <sstream>
+
+/// <summary>
+/// The GUID that identifies the notification activation callback.
+/// 68d0c89d-760f-4f79-a067-ae8d4220ccc1
+/// </summary>
+static constexpr winrt::guid CALLBACK_GUID{
+	0x68d0c89d, 0x760f, 0x4f79, {0xa0, 0x67, 0xae, 0x8d, 0x42, 0x20, 0xcc, 0xc1}
+};
+
+/// <summary>
+/// String representation of the callback GUID.
+/// </summary>
+const std::string CALLBACK_GUID_STR = "{68d0c89d-760f-4f79-a067-ae8d4220ccc1}";
```

Comment:
According the resources doc in  UpdateRegistry. It looks like every application should have a unique GUID, so the GUID here should be configurable. Maybe change the GUID to be passed from the initialization parameter instead?

Reply:
Gotcha, will fix.